### ISSUE TITLE
Switch to git for website

### ISSUE THREAD

**Issue by asfimport** [apache/parquet-java#2377]
Using git for the website, rather than SVN, would allow website changes to be proposed by non-committers as pull requests and reviewed in Github. For more, see:

 

<https://blogs.apache.org/infra/entry/git_based_websites_available>

<https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories>

<https://www.apache.org/dev/project-site.html>

 

 

**Reporter**: [Jim Apple](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jbapple) / @jbapple
**Assignee**: [Gabor Szadovszky](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gszadovszky) / @gszadovszky
#### Related issues:
- [Automate site generation](https://issues.apache.org/jira/browse/PARQUET-1686) (relates to)

<sub>**Note**: *This issue was originally created as [PARQUET-1675](https://issues.apache.org/jira/browse/PARQUET-1675). Please see the [migration documentation](https://issues.apache.org/jira/browse/PARQUET-2502) for further details.*</sub>

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1675?#comment-16962827) / @gszadovszky:
The transfer is done, <https://github.com/apache/parquet-site> is available and in sync with our website. Currently, the process is similar than we have before: manually generate the content and push to github.

Next step is use the [Pelican CMS System](https://blog.getpelican.com/) as described at [.asf.yaml features for git repositories](https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories#id-.asf.yamlfeaturesforgitrepositories-StaticwebsitecontentgenerationviaPelicanCMS).

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1675?#comment-16962980) / @gszadovszky:
Seems that Pelican works a bit differently than middleman. I am not sure if Pelican is the best way to step forward and currently I don't have time to work on that. Closing this issue (as we already moved to git) and creating another one to give the opportunity to another volunteer to work on automating the site generation.
